src/mealDb.js

```diff
@@ -0,0 +1,180 @@
+import { Displaylikes, like } from './likeSection.js';
+
+const popup = async () => {
+  const header = document.querySelector('header');
+  const main = document.querySelector('.meals-section');
+  const footer = document.querySelector('footer');
+  const btn = document.getElementsByClassName('comments');
+  const modal = document.querySelector('#modal');
+
+  const dataModal = async (id) => {
+    const mealapiurl = `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`;
+    const meals = await fetch(mealapiurl)
+      .then((res) => res.json())
+      .then((data) => data.meals);
+    meals.forEach((element) => {
+      if (element.idMeal === id) {
+        modal.innerHTML = `
+        <div class="modal">
+        <div class="meal-header justify-end d-flex">
+            <button class="close"><i class="fas fa-times"></i></button>
+        </div>
+        <div class="modal-header-img d-flex ">
+            <img class="modal-img" src="${element.strMealThumb}" alt="${element.strMeal}">
+            <h2 class="titme">${element.strMeal}</h2>
+        </div>
+        <div class="modal-description d-flex ">
+        <div class = "left">
+        <p> <span class="description-header">Category:</span> ${element.strCategory} <p>
+        <p> <span class="description-header">Ingredients:</span>
+        <ul id = "ingredientsUl">
+        </ul>
+        </span><p>
+        </div>
+        <div class = "right">
+        <p> <span class="description-header">Country:</span> ${element.strArea} <p>
+        <p> <span class="description-header">Instructions:</span> <p> <span class = "recipe-instrruction"> ${element.strInstructions}  > read more </button> </span>
+        </div>
+        </div>
+        <div class="comments-container">
+          <div class="all-comments text-center">
+            <h3>Comments <span id= "comments-count">0</span></h3>
+            <ul id="comments-ul" class = "d-flex justify-center flex-col">
+
+            </ul>
+          </div>
+          <div class = "text-center">
+            <h3> Add a comment </h3> 
+          </div>
+      
+    <div class="add-comments text-center d-flex justify-center">
+      <form action="POST">
+          <input name= "name" type="text" id="input-name" placeholder="Your name" required>
+          <textarea name="comment" id="comment" cols="30" rows="10" required></textarea>
+          <span id = "alert" class="text-start"> </span>
+          <button id="submit-comment" class="submit-comment" data="${element.idMeal}"> Submit</button>
+          
+      </form>
+    </div>
+    </div>
+      `;
+
+        const ingredientsUl = document.querySelector('#ingredientsUl');
+        const ingredients = () => {
+          const entries = Object.entries(element);
+          const ingredientsArray = entries
+            .filter(
+              ([key, value]) => key.startsWith('strIngredient') && value && value.trim(),
+            )
+            /* eslint-disable-next-line */
```

Comment:
- [ ] [OPTIONAL] It would be great if, instead of disabling eslint for some lines, you can find a way to fix the error produced by the linter itself, as part of good practice. However, as this is not a major problem I am not stopping your progress because of this, but take notes for the upcoming projects!

Reply:
@Lordkaito. Thanks for the suggestion. We'll keep that in mind and make the corrections